{
  underTest.startup();
  expectedException.expect(IllegalStateException.class);
  expectedException.expectMessage("startup() can not be called multiple times");
  underTest.startup();
}
